uiowa/uiowa
# [ITS] Allow alerts to be replicated

Resolves #7433 





# How to test



```
ddev blt ds --site=its.uiowa.edu && ddev drush @its.local uli
```

Replicate an alert, see that it works

Try syncing another site and make sure it syncs cleanly. If you sync facilities, you should be able to replicate alerts
